package.json

```diff
@@ -25,9 +25,10 @@
   "license": "MIT",
   "scripts": {
     "lint": "xo",
-    "test": "npm run test:node && npm run test:browser",
+    "test": "npm --node run test:node && npm run test:browser && npm --electron run test:electron",
```

Comment:
`--node` and `--electron` get passed to test as `process.env.npm_config_node` and `process.env.npm_config_electron`.